src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js

```diff
@@ -10,6 +10,8 @@ import ApiAuthorizationRoutes from './components/api-authorization/ApiAuthorizat
 import { ApplicationPaths } from './components/api-authorization/ApiAuthorizationConstants';
 ////#endif
 
+import './custom.css'
```

Comment:
Don’t name this custom. Look up the common react convention for this and if not either name it site.css or app.css to match the file.

Reply:
Wait, seems like custom was what we were doing. Why do we change it from scss to css (as I expect scss to be processed at build)

Reply:
The scss file that existed before was both empty and unused. If someone had tried to import it as I do here they would have gotten an error like `You attempted to import ../scss/custom.scss which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.`. If we try just moving the scss file within the src directory we instead get `To import Sass files, you first need to install node-sass.
Run `npm install node-sass` or `yarn add node-sass` inside your workspace.`. I figured that it wasn't worth it to add an additional OSS dependency when the styles we're applying makes no use of the additional features from scss, so I made it a css file.